.github/workflows/main.yml

```
@@ -13,29 +13,39 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
   
-before_install:
-  
-  
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   test:
     # The type of runner that the job will run on
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-#        os: [ubuntu-18.04, macos-10.15, windows-2019]
+#        os: [ubuntu-18.04, macos-10.15]
         os: [ubuntu-18.04 ]
+#        compiler: [gfortran-8, gfortran-9]
+        compiler: [gfortran-9]
+    env:
+      FC: ${{ matrix.compiler }}
     steps:
     - name: Getting repository
       uses: actions/checkout@v2
     - name: Install dependencies
       run: |
         sudo apt-get install gfortran libblas-dev liblapack-dev m4
+        git clone https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
```

```suggestion
        git clone -b v4.1.15 https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
```
If possible it's generally a good idea to checkout a specific tag rather than master, to avoid your build potentially breaking when master is updated